packages/core/src/components/button/_common.scss

```diff
@@ -260,16 +271,20 @@ $button-intents: (
   }
 
   &:disabled {
-    box-shadow: none;
-    background-image: none;
-    color: rgba($white, 0.3);
+    @include pt-dark-button-intent-disabled();
   }
 
   .pt-button-spinner .pt-spinner-head {
     stroke: $dark-progress-head-color;
   }
 }
 
+@mixin pt-dark-button-intent-disabled() {
```

Comment:
I'm not crazy about adding __four__ new mixins to this file, but I get it.  I wonder if it's reasonable to just add some params to the first disabled mixin for `color` and `background`?